pkg/pillar/cas/containerd.go

```diff
@@ -695,26 +695,32 @@ func (c *containerdCAS) PrepareContainerRootDir(rootPath, reference, _ string) e
 		logrus.Errorf(err.Error())
 		return err
 	}
-	clientImageSpecJSON, err := getJSON(clientImageSpec)
+	clientImageSpecJSON, err := json.Marshal(clientImageSpec)
```

Comment:
I think it was just applying the standard indentation anyway (4 spaces per indent), which is also applied by the `Marshal` function, but also there is no need to do it for a machine readable file

Reply:
Sure? The default behavior seems is not the case:  https://go.dev/play/p/r1CoIzFLpRZ
But anyway, would be great if you add a few words why it is removed.

Reply:
sorry, you are right, I put the MarshalIndent back now, because the main reason for refactoring was to get rid of the unnecessary function `getJSON`, indenting was not the reason